### ISSUE TITLE
Domain focus and not only entire forest

### ISSUE THREAD

**Issue by red-erik** [ryanries/ADReplStatus#12]
Hello,
thank you for your great work.
It would be of some help, into very large and complex environment. to be able to select only a specific
subdomain and not the entire forest.

Regards,
Red.

**Comment by ryanriesMSFT** [comment]
@red-erik Yes this is a great idea and something we should definitely implement in the future.
